src/Symfony/Component/Form/Extension/Core/DataTransformer/DateTimeToArrayTransformer.php

```diff
@@ -170,9 +170,9 @@ public function reverseTransform($value)
                 empty($value['year']) ? $this->referenceDate->format('Y') : $value['year'],
                 empty($value['month']) ? $this->referenceDate->format('m') : $value['month'],
                 empty($value['day']) ? $this->referenceDate->format('d') : $value['day'],
```

Comment:
can you please ensure we have a test case using `0` for year+month+day?

Reply:
Added it as test for DateType.